nextflow_schema.json

```
@@ -726,9 +726,204 @@
+                    "type": "boolean",
+                    "description": "Turn off duplicate removal.",
+                    "help_text": "Turn off duplicate removal. By default, reads that are an exact copy (i.e. same start, stop coordinate and exact sequence match) will be removed as it is considered a PCR duplicate.\\n\\nOnly when `--metagenomics_profiling_tool malt` is also supplied.\n\n> Sets tool parameter(s):\n> - MaltExtract: `--dupRemOff`",
+                    "fa_icon": "fas fa-copy"
+                },
+                "metagenomics_maltextract_matches": {
+                    "type": "boolean",
+                    "description": "Turn on exporting alignments of hits in BLAST format.",
+                    "help_text": "Export alignments of hits for each node in BLAST format. By default turned off.\\n\\nOnly when `--metagenomics_profiling_tool malt` is also supplied.\n\n> Modifies tool parameter(s):\n> - MaltExtract: `--matches`",
+                    "fa_icon": "fas fa-equals"
+                },
+                "metagenomics_maltextract_megansummary": {
+                    "type": "boolean",
+                    "description": "Turn on export of MEGAN summary files.",
+                    "help_text": "Export 'minimal' summary files (i.e. without alignments) that can be loaded into [MEGAN6](https://doi.org/10.1371/journal.pcbi.1004957). By default turned off.\\n\\nOnly when `--metagenomics_profiling_tool malt` is also supplied.\n\n> Sets tool parameter(s):\n> - MaltExtract: `--meganSummary`"
+                },
+                "metagenomics_maltextract_minpercentidentity": {
+                    "type": "number",
+                    "default": 85,
+                    "description": "Minimum percent identity alignments are required to have to be reported as candidate reads. Recommended to set same as MALT parameter.",
+                    "help_text": "Minimum percent identity alignments are required to have to be reported. Higher values allows fewer mismatches between read and reference sequence, but therefore will provide greater confidence in the hit. Lower values allow more mismatches, which can account for damage and divergence of a related strain/species to the reference. Recommended to set same as MALT parameter or higher. Default: `85`.\\n\\nOnly when `--metagenomics_profiling_tool malt` is also supplied.\n\n> Modifies tool parameter(s):\n> - MaltExtract: `--minPI`"
+                },
+                "metagenomics_maltextract_usetopalignment": {
+                    "type": "boolean",
+                    "description": "Turn on using top alignments per read after filtering.",
+                    "help_text": "Use the best alignment of each read for every statistic, except for those concerning read distribution and coverage. Default: off.\\n\\nOnly when `--metagenomics_profiling_tool malt` is also supplied.\n\n> Sets tool parameter(s):\n> - MaltExtract: `--useTopAlignment`",
```

```suggestion
                    "help_text": "Use the best alignment of each read for every statistic, except for those concerning read distribution and coverage. \\n\\nOnly when `--metagenomics_profiling_tool malt` is also supplied.\n\n> Sets tool parameter(s):\n> - MaltExtract: `--useTopAlignment`",
```

Default not needed in text, check everywhere

---

done